src/django/api/tests/test_source_contributor_change.py

```
@@ -0,0 +1,252 @@
+        # Verify initial state
+        self.assertEqual(
+            self.extended_field.contributor, self.original_contributor)
+        self.assertEqual(
+            self.extended_field_2.contributor, self.original_contributor)
+
+        # Create mock request and form
+        request = Mock()
+        request.user = self.admin_user
+        messages_list = []
+
+        def mock_message_user(req, msg, level=None):
```

Ruff (0.13.2)

Unused function argument: `req`

(ARG001)